### PR TITLE
Add forgotten CHANGELOG and prepare 0.3.1 release

### PR DESCRIPTION
**Pull request by therealprof**
Signed-off-by: Daniel Egger <daniel@eggers-club.de>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+
+### Fixed
+
+- Fixed confusion caused by `build.rs` in the project generating an virtual
+  artifact resulting in `error: Can only have one matching artifact but found
+  several`
+
+### Changed
+
+- Allow the combination of  `features` `no-default-features` and `all-features` flags
```
Comment (jonas-schievink) [inline CHANGELOG.md:20]:
```suggestion
- Allow the combination of `--features` `--no-default-features` and `--all-features` flags
```

**Comment by therealprof** [comment]
@jonas-schievink Thanks for the correction.

**Comment by jonas-schievink** [comment]
Oh, right

**Comment by therealprof** [comment]
bors hello?

**Pull request merged**